main.go

```diff
@@ -73,10 +102,40 @@ func main() {
 		os.Exit(1)
 	}
 
+	if region == "" {
+		setupLog.Error(err, "--region flag not provided")
```

Comment:
Shouldn't it also support discovery via ec2 metadata API?

Reply:
Yes, but that would make running locally `make run` problematic

Reply:
We will leverage environment variable - I will fix it